pact/fixed-quote-policy-royalty.pact

```diff
@@ -0,0 +1,192 @@
+(namespace (read-msg 'ns))
+
+(module fixed-quote-policy-royalty GOVERNANCE
+
+  @doc "Policy for fixed issuance with simple quoted sale."
+
+  (defcap GOVERNANCE ()
+    (enforce-guard (keyset-ref-guard 'marmalade-ns-admin )))
+
+  (implements kip.token-policy-v1_DRAFT3)
+  (use kip.token-policy-v1_DRAFT3 [token-info])
+
+  (defschema policy-schema
+    fungible:module{fungible-v2}
+    creator:string
+    creator-guard:guard
+    mint-guard:guard
```

Comment:
I think `creator-guard` can also be `mint-guard`, no?

Reply:
`creator-guard` is also used for the account guard in the designated fungible that will receive the fund, so I think `mint-guard` can serve it's own purpose? If the creator wants to use the same guard, they can put the same key in the guard, but they might not want to unify them .